app/src/main/java/ch/epfl/favo/viewmodel/FavorViewModel.java

```diff
@@ -276,6 +278,20 @@ public void setFavorValue(Favor favor) {
     return observedFavor;
   }
 
+
+  @Override
+  public LiveData<User> getObservedUser() {
+    getUserRepository()
+            .getCurrentUserReference(currentUserId)
+            .addSnapshotListener(
+                    MetadataChanges.EXCLUDE,
```

Comment:
can change MetadataChanges to INCLUDE which might make it react faster when seeing local changes

Reply:
fixed